src/igor/sub.go

```diff
@@ -82,6 +84,33 @@ func runSub(cmd *Command, args []string) {
 	var newSched []TimeSlice    // the new schedule
 	format := "2006-Jan-2-15:04"
 
+	// parse duration requested
+	var days int = 0
+	duration = 0
+	nanoseconds, err := time.ParseDuration(subT)
+	if err != nil {
+		// Check for a number of days in the argument
+		var dInd int
+		if dInd = strings.Index(subT, "d"); dInd >= 0 {
```

Comment:
If a variable is only used in the `if`, you can declare it inline:

`if dInd := strings.Index(subT, "d"); dInd >= 0 {`

Reply:
Old habits :-)